src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const stateClone = { ...state };
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
checklist points 2, 3. use switch, case, default instead if

Reply:
good job, but change the code according to the checklist for this and the previous task

Reply:
Thank you!